python/lsst/ts/electrometer/commander.py

```diff
@@ -0,0 +1,72 @@
+import asyncio
+import logging
+from lsst.ts import tcpip
+
+
+class TCPIPCommander:
```

Comment:
Maybe just name this class to be `Commander` or rename the `commander.py` to `tcpip_commander.py` for the consistency.